### ISSUE TITLE
midterm report feedback

### ISSUE THREAD

**Issue by Tdrinker** [apiando/fake-news-detection#2]
you essentially didn't create a train/test/val split. Yours is only train/test. This has the potential danger or overfitting to the test set.

Also, bert is the most basic version of the bert family. You can try better ones like RoBERTa or other SOTA models and tokenizers to see if you get better results. 

and the choice of the -uncased tokenizer needs to be justified. why uncased? isn't it true that the capitalized letters overuse might also be a potential feature of fake news? it's worth exploring. You can compare cased and uncased.

also please quantitatively report your performances in the result as well as the baselines (for example, random guessing).

overall: 8/10.

**Comment by Tdrinker** [comment]
also worth doing some visualization. what are the key contributing features?
